webapp/src/main/java/life/qbic/views/ErrorMessage.java

```diff
@@ -0,0 +1,68 @@
+package life.qbic.views;
+
+import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+
+/**
+ * <b> An ErrorMessage component which shows an error message with a title and a detailed
+ * description. </b>
+ *
+ * @since 1.0.0
+ */
+public class ErrorMessage extends Composite<VerticalLayout> {
+
+  private final String descriptionText;
+  private final String titleText;
+
+  private Span titleSpan;
+  public Span titleTextSpan;
+
+  private Div descriptionDiv;
+  public Span descriptionTextSpan;
```

Comment:
I am a bit confused about which fields are public and which are not and why there is a difference. Can someone explain? :)

Reply:
The fields that contain the text are public so they could be accessed e.g to get the content. 
The idea was to hide the other elements since they are only containers for the text elements

Reply:
I think then you should introduce setters. this mixture looks strange